lib/events.ts

```diff
@@ -1,11 +1,9 @@
 import { CoreKernel } from '@balena/jellyfish-core';
-import { getLogger, LogContext } from '@balena/jellyfish-logger';
-import { JSONSchema } from '@balena/jellyfish-types';
-import {
-	ExecuteContract,
-	ExecuteContractDefinition,
-} from '@balena/jellyfish-types/build/queue';
-import { PostResults, PostOptions } from './consumer';
+import { getLogger } from '@balena/jellyfish-logger';
+import type { LogContext } from '@balena/jellyfish-logger';
+import { JsonSchema } from '@balena/jellyfish-types';
```

Comment:
This can be `import type` too. Take a look here for more details: https://devblogs.microsoft.com/typescript/announcing-typescript-3-8-beta/#type-only-imports-exports.

Reply:
Thanks for catching that, just missed adding in the keyword.